Resources/Prototypes/Catalog/Fills/Items/belt.yml

```
@@ -191,10 +191,12 @@
   components:
   - type: StorageFill
     contents:
-        - id: WeaponRevolverInspector
+        #- id: WeaponRevolverInspector #Sunrise-edit #Moved to loadout
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)